src/app/checkout/step-3/step-3.tpl.html

```diff
@@ -13,10 +13,14 @@
           id="changeContactButton"
           class="btn btn-default btn-panel-head pull-right"
           ng-click="$ctrl.changeStep({newStep: 'contact'})"
+          aria-describedby="contact-info"
           translate
         >
           {{'CHANGE'}}
         </button>
+        <span id="contact-info" class="sr-only">
```

Comment:
This definitely works! But what do you think about having an aria-label instead that says something like "Update contact info". I'm thinking that hearing that read sounds a little clearer than hearing, "Change [pause] Click Change to update your contact information".